src/Extraction/GitDataExtractor/GitDataExtractor/appsettings.json

```diff
@@ -1,4 +1,5 @@
 {
     "RepositoryDirectory": "C:\\repos\\infvis",
-    "CommitHistoryFilePath": "./history.json"
+    "CommitHistoryFilePath": "./history.json",
+    "AggregateFilePath": "./aggregate.json"

```

Comment:
Please try to execute and check the result. Does it fit to your needs?
Then I'll run it for the big repo and upload somewhere.

Reply:
Cool!
Ran it and with the data I created a first draft version of the treemap that renders a single aggregated virtual commit.
To represent the state at any given time I still have to do some processing of the data to accumulate everything up until this point. But it should be doable as everything is there.

I'm basing my further code changes on this PR. So please go ahead and merge :)